.idea/codeStyles/Project.xml

```diff
@@ -44,6 +44,9 @@
     <XML>
       <option name="XML_LEGACY_SETTINGS_IMPORTED" value="true" />
     </XML>
+    <editorconfig>
```

Comment:
you will probably have to re-enable this in your settings to get rid of this change